quinn/examples/client.rs

```diff
@@ -105,7 +108,7 @@ async fn run(options: Opt) -> Result<()> {
     let host = options
         .host
         .as_ref()
-        .map_or_else(|| url.host_str(), |x| Some(x))
+        .map_or_else(|| url.host_str().map(strip_ipv6_brackets), |x| Some(x))
```

Comment:
nit: maybe reuse the result from doing this above?

Reply:
Sounds good. Incidentally, it looks like that error handling case was superfluous, because the function the function already began by unwrapping `url.host_str()`. So it's a bit simpler now overall.